guide/src/StyleGuide.js

```diff
@@ -141,7 +141,7 @@ export default React.createClass({
             main: {
                 background: "whitesmoke",
                 width: "100%",
-                margin: "50px 0px 50px 250px", 
+                margin: "50px 0px 0px 250px", 
```

Comment:
This should be in a different PR.
I can explain why but the value should be `50px auto 0px`

If the issue you were seeing was the background color, a solution would be to move the background color to the parent.